<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteTool</class>
 <widget class="QDialog" name="NoteTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>54</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignVCenter">
    <widget class="QPushButton" name="newBookBtn">
     <property name="text">
      <string>新建笔记</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PullDownList" name="pull">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignVCenter">
    <widget class="QPushButton" name="readBtn">
     <property name="text">
      <string>阅读</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignVCenter">
    <widget class="QPushButton" name="outBookBtn">
     <property name="text">
      <string>导出笔记</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PullDownList</class>
   <extends>QFrame</extends>
   <header>view/pulldownlist/pulldownlist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
